### PR TITLE
Add missing changelog entries from previous PRs

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.0.0
+-----
+
+ Removed support to load translation resources from the legacy directories `src/Resources/translations/` and `src/Resources/<BundleName>/translations/`
```
Comment (yceruto) [inline src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md]: from https://github.com/symfony/symfony/pull/31668

File: src/Symfony/Bundle/TwigBundle/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.0.0
+-----
+
+ * updated default value for the `strict_variables` option to `%kernel.debug%` parameter
```
Comment (yceruto) [inline src/Symfony/Bundle/TwigBundle/CHANGELOG.md]: from https://github.com/symfony/symfony/pull/31664

File: src/Symfony/Component/Form/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.0.0
+-----
+
+ * removed the `ChoiceLoaderInterface` implementation in `CountryType`, `LanguageType`, `LocaleType` and `CurrencyType`
```
Comment (yceruto) [inline src/Symfony/Component/Form/CHANGELOG.md]: from https://github.com/symfony/symfony/pull/31665

File: src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md (hunk 1 above)
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md]: missing` *` :)

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**